pkg/prompb/remote.pb.go

```diff
@@ -68,7 +68,6 @@ type WriteRequest struct {
 	XXX_sizecache        int32        `json:"-"`
 }
 
```

Comment:
I'd maybe leave a comment here that the Reset function was moved to custom.pb.go. Same for all places it moved